module/users/helper.js

```diff
@@ -300,25 +351,33 @@ module.exports = class UserHelper {
                             solutionData => solutionData.externalId === solution.externalId
                         );
 
-                        if( solutionIndex < 0 ) {
-
-                            let solutionOrObservationInformation = 
-                            users[user].isObservation ? 
-                            _observationInformation(
-                                program,
-                                users[user],
-                                solution
-                            ) : _solutionInformation(program,solution); 
-                            
-                            solutionOrObservationInformation["entities"] = [];
-                            result[programIndex].solutions.push(
-                                solutionOrObservationInformation
-                            );
+                        if (solutionIndex < 0) {
+
+                            let solutionOrObservationInformation =
```

Comment:
@deepakr123 - We should not pass surveyInformation as observationInformation

Reply:
resolved